### ISSUE TITLE
After using transform to hide, the text style will be lost after changing the window size

### ISSUE THREAD

**Issue by yaoguoba** [highcharts/highcharts#18282]
#### Expected behaviour
no change in font size
![IMG_5085](https://user-images.githubusercontent.com/18393410/211999800-212354a3-43de-4d48-9624-93307847c4f1.jpg)

#### Actual behaviour
font size gets larger
![IMG_5084](https://user-images.githubusercontent.com/18393410/211999832-c70755bb-47f3-4265-b67a-12c9dd7e1705.jpeg)

#### Live demo with steps to reproduce
https://jsfiddle.net/aoq4xu89/7

Click toggle button -> resize the window size -> Click toggle button again

You will see that the text is getting larger.
At this time, if you change the window size again, the font size will return to normal.

#### Product version
9+

#### Affected browser(s)
Chrome, Edge
(Safari,Firefox is working fine)

**Comment by bm64** [comment]
Hello @yaoguoba.

I've tried to recreate the bug and it seems to occur on my side only in case when scale property is being used to hide the chart and browser zoom level is different than 100%. If it's less than 100% the text seem to shrink, if it's more it seems to grow like in your example. It looks like an edge case to me, so for now I'm leaving this ticket as `undecided` until we gather more feedback from users.
When it comes to a workaround, if that's possible try to use for example display property to hide the chart: https://jsfiddle.net/BlackLabel/w23cexz8/

**Comment by wixdonkey** [comment]
I ran into this problem as well. At that time, I didn't get the root cause and fix it. Then I got a complain from client, so bad. :(
Is there any plan to fix this in highchart orginally. That's would be great to have this fix.
Anywah, it's so great that I can find the details about this problem. Thanks all guys!

**Comment by yaoguoba** [comment]
Hi @bm64, thanks for your reply. I did test display:none without any problems. But sometimes we will add some animation effects when displaying and hiding. That's why I found 'transform:scale' is the root cause. And I think this kind of animation effect should be very common.

**Comment by wixdonkey** [comment]
> 
Yes, I think so. I used the company custom component "Collapse" as the container of highchart. Used to display two different sets of data.
When customers check one set of data, they often collapsed another set. And then, sometimes they would adjust the size of the window. When the hidden data is opened again, the style has been broke. Then they have to resize the windows to make the chart display well.

**Comment by bm64** [comment]
@wixdonkey , @yaoguoba thanks for your feedback.

**Comment by bm64** [comment]
Hello @yaoguoba unfortunately I can't provide you with any update at this moment. The workaround seems to work, and we don't have any estimated time for fixing it yet.
